neo/Persistence/LevelDB/DbTrieStore.cs

```diff
@@ -0,0 +1,53 @@
+using Neo.IO.Data.LevelDB;
+using Neo.Trie;
+using Neo.Trie.MPT;
+using Neo.Ledger;
+
+namespace Neo.Persistence.LevelDB
+{
+    public class DbTrieStore : IKVStore
+    {
+        private readonly DB db;
+        private readonly ReadOptions options;
+        private readonly WriteBatch batch;
+        private readonly byte prefix;
+
+        private readonly byte[] ROOT_KEY = Blockchain.GenesisBlock.Hash.ToArray();
```

Comment:
It's the same as https://github.com/neo-project/neo/pull/1803/files#diff-d9b8054e080c526156aa17afc3420f6cR14

Reply:
Should be the same. Because they use the same store.

Reply:
But it could be shared?

Reply:
Oh, I see.

Reply:
Fixed in #1814